tests/test_quantity.py

```diff
@@ -0,0 +1,308 @@
+import pytest
+
+import ansys.fluent.core.quantity as q
+
+
+def test_viscosity():
+    v = q.Quantity(1, "P")  # poise
+    conversion_output = v.to("kg m^-1 s^-1")
+    assert conversion_output.__float__() == 0.1
+
+
+def test_dynamic_viscosity():
+    vd = q.Quantity(1, "Pa s")
+    conversion_output = vd.to("P")
+    assert conversion_output.__float__() == 10.0
```

Comment:
Why can't this be `conversion_output == 10.0`?

Reply:
Because ```conversion_output``` is quantity instance with value and magnitude. 
Now we have derived the Quantity from the float, hence we have used ```conversion_output.__float__() == 10.0```  

![image](https://user-images.githubusercontent.com/106588300/187492278-c9b0fdc5-2f0c-450e-8f8d-79ed5493cdbe.png)

Reply:
OK, that's fine, but it seems a little strange to me to call the `__float()__` method directly like that.

Reply:
Then should we use ```conversion_output.real == 10.0``` while testing?

Reply:
I don't see why not. 

You might also consider implementing  `__eq__()` then this might work too:

`assert vd == q.Quantity(1, 'Pa s)`

Reply:
Yes, sure. We will implement above method as well. Thank you for the suggestion.

Reply:
![image](https://user-images.githubusercontent.com/106588300/187503816-b9e2b172-886a-4757-b8b4-e94934087618.png)

Reply:
Test also passed for above method - 

![image](https://user-images.githubusercontent.com/106588300/187504535-787fdda0-da7c-41fe-87ca-83fe092cf399.png)

![image](https://user-images.githubusercontent.com/106588300/187504673-f5d8daee-4b8a-4788-ae60-fdd6cbac9217.png)

Reply:
Replaced ```conversion_output.__float__()``` with ```conversion_output.real```. 
Added equal check method and it's test case.

Reply:
> OK, that's fine, but it seems a little strange to me to call the `__float()__` method directly like that. I guess if we are accessing the value I kind of preferred the previous interface, i.e. `conversion_output.value`.

Done. 

![image](https://user-images.githubusercontent.com/106588300/187832317-c8e7e81a-bc19-4c94-a97e-d26ffd3ae10b.png)

Now we can access value of quantity instance using following ways.

![image](https://user-images.githubusercontent.com/106588300/187832573-467691b5-b9c6-4bc4-ba2e-933f1f258a9d.png)